### DISCUSSION TITLE
Gen Z

### DISCUSSION THREAD

**Discussion by yulisaffa** [networkx/networkx#7559] [Q&A]
### Environment

Python version:
NetworkX version:

**Comment by yulisaffa** [comment]
can mine data with Gen Z keywords. I want to know how Gen Z's social behavior is on social media

**Comment by Peiffap** [comment]
This is probably better suited to be a [discussion](https://github.com/networkx/networkx/discussions).
It could also use some additional information and clarification. What do you mean by Gen Z's social behavior? Do you have a specific data set in mind? Perhaps we will be better able to help you if you can figure out what the answers to those questions are.

**Comment by rossbar** [comment]
I'm going to go ahead and close this for now, as it strikes me as a general data-mining question rather than anything specific to the NetworkX library. If there are followup questions related specifically to networkx, please feel free to open with the appropriate context!
